pkgs/applications/version-management/sr-ht/default.nix

```diff
@@ -0,0 +1,31 @@
+{ python36, callPackage }:
+
+# srht officially uses python 3.7, however, we prefer python 3.6
+# https://lists.sr.ht/~sircmpwn/sr.ht-admins/<20180807190137.GA20634%40miku>
+# https://github.com/NixOS/nixpkgs/pull/54425#discussion_r249489825
+
+let
+  fakePython = python36.override {
+    packageOverrides = self: super: {
+      redis = super.redis.overridePythonAttrs (oldAttrs: rec {
+        version = "2.10.6";
+
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "03vcgklykny0g0wpvqmy8p6azi2s078317wgb2xjv5m2rs9sjb52";
+        };
+      });
+    };
```

Comment:
```suggestion
    };
    core-sr-ht = self.callPackage ./core.nix { inherit fetchNodeModules; };
    meta-sr-ht = self.callPackage ./meta.nix { };
```

Reply:
If I interpret this correctly, switch `buildPythonApplication` to `buildPythonPackage`, find a way to index the new `python` with the modules and then use `withPackages` in it?

Reply:
Exactly. And you can make `python` know about the modules by putting them in `packageOverrides`.

Reply:
All of them? I had the impression only core and meta would ever be used as modules. And no, I really don't think there's any use in having them in `python-packages.nix`.

Reply:
Yeah, since it seems that when `gunicorn` is run, it imports the module from each package.

Reply:
Ok, then just use `buildPythonPackage` and add a comment on why it's needed.